imports/server/googleClientRefresher.ts

```diff
@@ -51,16 +52,24 @@ class GoogleClientRefresher {
       service: "google",
     });
     this.oauthCredentialCursor = Settings.find({ name: "gdrive.credential" });
-    void this.oauthConfigCursor.observeAsync({
-      added: (doc) => this.updateOauthConfig(doc),
-      changed: (doc) => this.updateOauthConfig(doc),
-      removed: () => this.updateOauthConfig(undefined),
-    });
-    void this.oauthCredentialCursor.observeAsync({
-      added: (doc) => this.updateOauthCredentials(doc),
-      changed: (doc) => this.updateOauthCredentials(doc),
-      removed: () => this.clearOauthCredentials(),
-    });
+    this.oauthConfigCursor
+      .observeAsync({
+        added: (doc) => this.updateOauthConfig(doc),
+        changed: (doc) => this.updateOauthConfig(doc),
+        removed: () => this.updateOauthConfig(undefined),
+      })
+      .catch((error) => {
+        Logger.error("Failed observing oauth config cursor", { error });
+      });
+    this.oauthCredentialCursor
+      .observeAsync({
+        added: (doc) => this.updateOauthCredentials(doc),
+        changed: (doc) => this.updateOauthCredentials(doc),
+        removed: () => this.clearOauthCredentials(),
+      })
+      .catch((error) => {
+        Logger.error("Failed observing oauth credential cursor", { error });
+      });
```

Comment:
Given that `GoogleClientRefresher` is a singleton and this constructor only runs at startup, what do you think about instead moving this to an `async init()` similar to the Discord client refresher (or a `static async create` a la the mediasoup SFU)?